mqtt/forwarder.go

```diff
@@ -37,7 +38,7 @@ func NewForwarder(topic string, logger log.Logger) Forwarder {
 }
 
 func (f forwarder) Forward(id string, sub messaging.Subscriber, pub messaging.Publisher) error {
-	return sub.Subscribe(id, f.topic, handle(pub, f.logger))
+	return sub.Subscribe(context.Background(), id, f.topic, handle(pub, f.logger))
```

Comment:
```suggestion
	return sub.Subscribe(context.TODO(), id, f.topic, handle(pub, f.logger))
```

Reply:
I think TODO context is better suited for tests

Reply:
Not really

> Code should use context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter).

Reply:
Not really

> Code should use context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter).